src/routes/auth.ts

```diff
@@ -0,0 +1,29 @@
+import { RequestHandler } from 'express';
+
+import { resinApi } from '../platform';
+import { handleHttpErrors } from '../platform/errors';
+
+export const getUserPublicKeys: RequestHandler = (req, res) => {
```

Comment:
Public keys are, well, public. We might as well just drop the newly added permissions, which are far too liberal (grants the device access to all users on the platform), and have this endpoint use `root` privileges to fetch public keys of any user.